Prox/Prox/PlaceDetails/PlaceDetailsIconInfoView.swift

```diff
@@ -32,15 +32,29 @@ class PlaceDetailsIconInfoView: UIView {
         return view
     }()
 
+    lazy var forwardArrowView = UIImageView(image: UIImage(named: "icon_forward"))
```

Comment:
I would prefer it is the `forwardArrowView` was a generic `accessoryItemView` or something, and the `PlaceDetailsCardView` sets the image depending on whether or not it is required, rather than the image being fixed inside the `PlaceDetailsIconInfoView` (like we are ding for the `iconView`. Doing it in the `PlaceDetailsIconInfoView` feels like it removes the generic nature of the view itself.

Also, it would remove the need to hide or show the `forwardArrowView`. It either has an icon or it doesn't.